CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and **Merged pull requests**. Critical items to know are:
 The versions coincide with releases on pypi.
 
 ## [0.0.x](https://github.com/singularityhub/singularity-compose/tree/master) (0.0.x)
+ - version 2.0 of the spec with added network and exec options (0.1.0)
```

Comment:
with added fakeroot network, start, exec, and run options

Reply:
Updated. I took a look at the 2 spaces - and it happens because there aren't run args. I could do a tweak to fix it, but because spaces aren't important for the command, I don't see  a reason that it's not ok.